google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/debugger/BreakpointComparer.java

```diff
@@ -75,14 +77,16 @@ public int compare(Breakpoint o1, Breakpoint o2) {
       }
       return s1.getPath().compareTo(s2.getPath());
     }
-    Date d1 = BreakpointUtil.parseDateTime(o1.getFinalTime());
-    Date d2 = BreakpointUtil.parseDateTime(o2.getFinalTime());
-    if (d1 == null) {
+
+    Date d1, d2;
+    try {
+      d1 = ISODateTimeFormat.dateTime().parseDateTime(o1.getFinalTime()).toDate();
```

Comment:
Yes, this was done in purpose, though. If the first timestamp fails to parse, it's because the server is crazy again and the second timestamp will almost surely also fail to parse.